docs/topic-guides/system-concepts/scheduling.txt

```diff
@@ -14,6 +24,21 @@ As new experiments become active or the resource demand of an active
 experiment changes, the scheduler will adjust how slots are allocated to
 experiments as appropriate.
 
-Scheduling behavior can be configured via the ``resources`` section of
-the experiment config file; see :ref:`experiment-configuration` for
-details.
+**********
+ Priority
+**********
+
+The master allocates cluster resources (slots) to active tasks based on
+their priority. While tasks of higher priority (lower priority number)
+are pending, no lower priority tasks will be scheduled. For instance, if
+tasks with priorities of five and forty-two are pending, the latter will
+not be scheduled until the prior has been. By default the priority
+scheduler will not preempt lower priority tasks in favor of higher
+priority tasks. If preemption is enabled, in scenarious where a higher
+priority task is pending and can not be scheduled, the scheduler will
```

Comment:
non-blocking: I think this can be simplified to "If preemption is enabled, the scheduler will attempt to schedule pending tasks by preempting lower priority ones, in order of lowest priority first".

Reply:
"cannot"